lib/loga/sidekiq6/job_logger.rb

```diff
@@ -0,0 +1,50 @@
+require 'sidekiq/job_logger'
+
+module Loga
+  module Sidekiq6
+    class JobLogger < ::Sidekiq::JobLogger
+      EVENT_TYPE = 'sidekiq'.freeze
+
+      def call(item, _queue)
+        start = ::Process.clock_gettime(::Process::CLOCK_MONOTONIC)
+
+        yield
+
+        with_elapsed_time_context(start) do
+          loga_log(
+            message: "#{item['class']} with jid: '#{item['jid']}' done", item: item,
+          )
+        end
+      rescue Exception => e # rubocop:disable Lint/RescueException
+        with_elapsed_time_context(start) do
+          loga_log(
+            message: "#{item['class']} with jid: '#{item['jid']}' fail", item: item,
+            exception: e
+          )
+        end
+
+        raise
+      end
+
```

Comment:
Don't we want to support the same public interface as the Sidekiq5 version? Shouldn't we add `started_at` and `data` as well?

Reply:
The interface we need to support is the one of `::Sidekiq::JobLogger`. TBO no idea why the `started_at` and `data` are public in Sidekiq5.